bubbles/message_utils.py

```diff
@@ -1,19 +1,74 @@
-from typing import Any, Dict, Union
+from __future__ import annotations
 
+from typing import Any, Dict, Union, Callable, Optional, TYPE_CHECKING
+from slack_sdk.web.client import WebClient
 
-class MessageUtils:
-    def __init__(self, client, payload):
+if TYPE_CHECKING:
+    from bubbles.plugins import PluginManager
+
+
+class Payload:
+    """Payload class for everything a command needs."""
+    def __init__(
+            self,
+            client: WebClient=None,
+            slack_payload: dict=None,
+            say: Callable=None,
+            context: dict=None,
+            meta: PluginManager=None,
```

Comment:
I think this is not correctly formatted (but I'm not sure), did you run black?

Reply:
Oops I probably didn't.